### DISCUSSION TITLE
Gene annotation missing for variant in SEQR web interface

### DISCUSSION THREAD

**Discussion by fbelleau** [broadinstitute/seqr#3204] [Q&A] [answered]
**Describe the bug**
We have two local installation of most recent seqr docker image

REPOSITORY                                      TAG           IMAGE ID       CREATED         SIZE
gcr.io/seqr-project/seqr                        gcloud-prod   f98a977f619a   8 days ago      2.4GB

On one server #1 we have the gene annotation shown properly and on server #2 we do not have annotation and only the ensembl ID is shown.

Can you help me identify the missing annotation file on server 2 or a log file I can look into to find an explanation ?

**Link to page(s) where bug is occurring**

http://server_1/variant_search/variant/1-874778-GCCTCCCCAGCCACGGTGAGGACCCACCCTGGCATGATCCCCCTCATCA-G/family/F000017_01_129

http://server_2/project/R0005_fam129_v38_v2/saved_variants/variant/SV0000007_1939398_f000004_01_1

**Scope of the bug**
It is a server issue during the visualisation of variant.

**Screenshots**

Annotation on server #1

![image](https://user-images.githubusercontent.com/995409/217358330-5911e729-600f-49f7-89cc-306020b49882.png)

Annotation missing on server #2

![image](https://user-images.githubusercontent.com/995409/217358427-fcbf7f06-88b1-4d9b-9dd3-360b6d4d4ef5.png)

**Comment by hanars** [comment, marked as answer]
It looks like on server #2 the gene-level reference data is not properly populated. This should have happened automatically when the docker image was pulled, but you can manually rerun the command by running `docker-compose exec seqr python -u manage.py update_all_reference_data --use-cached-omim`

**Reply by fbelleau** [reply]
It as solved the problem thank you.
